Tests/Features/Page/error.feature

```diff
@@ -0,0 +1,19 @@
+@mink:selenium2 @alice(Page) @alice(ErrorPage) @reset-schema
+Feature: Page not found
+
+Background:
+    Given I maximize the window
+    And I am on homepage
+
+  Scenario: I cannot acces a non exitant page
+    And I am on "/fr/imaginary-page"
+    Then I should see "404 not found"
```

Comment:
I tried to fix this by searching for `404 Not Found` but it failed too: https://circleci.com/gh/paulandrieux/victoire/461 So it's not the solution here.